static/accounts/style.css

```diff
@@ -315,4 +315,20 @@ input {
 	.radio-input-label1{
 		margin-right: 20px;
 	}
+}
+
+#password_div{
+	display: table;
+	width: 100%;
+	margin-left: -12px;
```

Comment:
Instead of using hard-coded negative margins, you can just change the `text-align` property of table to `left`.

You can visualize things by putting a border around the elements.